file/sequence_file_reader.cc

```diff
@@ -48,53 +62,87 @@ IOStatus SequentialFileReader::Read(size_t n, Slice* result, char* scratch) {
     buf.Alignment(alignment);
     buf.AllocateNewBuffer(size);
 
-    Slice tmp;
-    uint64_t orig_offset = 0;
-    FileOperationInfo::StartTimePoint start_ts;
-    if (ShouldNotifyListeners()) {
-      orig_offset = aligned_offset + buf.CurrentSize();
-      start_ts = FileOperationInfo::StartNow();
+    while (buf.CurrentSize() < size) {
+      size_t allowed;
+      if (rate_limiter_priority != Env::IO_TOTAL && rate_limiter_ != nullptr) {
+        allowed = rate_limiter_->RequestToken(
+            buf.Capacity() - buf.CurrentSize(), buf.Alignment(),
+            rate_limiter_priority, nullptr, RateLimiter::OpType::kRead);
```

Comment:
nit: add comment nullptr like /* stats */ - ditto for other RequestToken() below in this PR.

Reply:
sg, added.